src/app.js

```diff
@@ -5,14 +5,70 @@ console.log("hallo daar!");
 async function fetchData () {
```

Comment:
Blijkbaar werkt het wel, maar zou de opening en sluitingshaken achter FetchData er gelijk aanplakken in plaats van een spatie

Reply:
Oh ja dat is handig inderdaad :)